CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+Contributing
+============
+
+If you're here, you would like to contribute to this project and you're really welcome!
+
+Bug reports
+-----------
+
+If you find a bug or a documentation issue, please report it or even better: fix it :). If you report it,
+please be as precise as possible. Please use the provided template when creating an issue. Your issue is 
+more likley to get fast feedback if you are clear and detailed in your report. disclosing the vulnerability.
+
+Feature requests
+----------------
+
+If you think a feature is missing, please report it or even better: implement it :). If you report it, describe the more
+precisely what you would like to see implemented and we will discuss what is the best approach for it. If you can do
```

Comment:
Missing feature and a stray more?
'... describe the feature as precisely as possible, including what you would like to see implemented...'